examples/zkapps/02-private-inputs-and-hash-functions/src/IncrementSecret.ts

```diff
@@ -4,22 +4,14 @@ import {
   state,
   State,
   method,
-  DeployArgs,
   Poseidon,
-  Permissions,
-  PublicKey,
-  PrivateKey,
 } from 'snarkyjs';
 
 export class IncrementSecret extends SmartContract {
   @state(Field) x = State<Field>();
 
-  deploy(args: DeployArgs) {
-    super.deploy(args);
-    this.setPermissions({
-      ...Permissions.default(),
-      editState: Permissions.proofOrSignature(),
-    });
+  init() {
+    super.init();
```

Comment:
since `init()` doesn't do anything else than call the base `init()`, you can remove it

Reply:
I was thinking it's worth keeping in just to illustrate initialisation. I find too much magic a little unhellpful in tutorials. But we do have Square.ts in Example 01 that shows a more realistic initialisation so maybe it's better to be concise here. So I'll remove it, thanks 😄